bin/test-browser.js

```diff
@@ -181,90 +136,62 @@ function BenchmarkReporter(runner) {
   });
 }
 
-function startTest() {
+async function startTest() {
 
-  console.log('Starting', client, 'on', testUrl);
+  console.log('Starting', browserEngine, 'on', testUrl);
 
-  var opts = {
-    browserName: client.browser,
-    version: client.version,
-    platform: client.platform,
-    tunnelTimeout: testTimeout,
-    name: client.browser + ' - ' + tunnelId,
-    'max-duration': 60 * 45,
-    'command-timeout': 599,
-    'idle-timeout': 599,
-    'tunnel-identifier': tunnelId
-  };
-  if (CHROME_BIN) {
-    opts.chromeOptions = {
-      binary: CHROME_BIN,
-      args: ['--headless', '--disable-gpu', '--no-sandbox', '--disable-setuid-sandbox']
-    };
-  }
-  if (FIREFOX_BIN) {
-    opts.firefox_binary = FIREFOX_BIN;
-  }
+  const browserImpl = playwrightBrowsers[browserEngine];
```

Comment:
Playwright has a class `Browser`.  `browserImpl.launch()` returns an instance of `Browser` at line 152, which is assigned to `const browser`, so that name's already taken.

Following the examples in the docs, calling the `Browser` instance `browser` seems to be idiomatic in playwright code.

+1 for `browserName` - this seems to be the standard name in e.g. https://playwright.dev/docs/api/class-fixtures#fixtures-browser-name

Any alternative suggestions to `browserImpl`?

Reply:
a js convention seems to be to have capitals for the class and small case for the instance. that would be my favorite, but ima also not opposed to use something like browserInstance or browserInst.

`const browser = Browser.launch()`

Reply:
I've eliminated the intermediate variable, so now there is just `browserName` and `browser`.